release.go

```diff
@@ -186,8 +163,63 @@ func (r *releaseRepository) Get(release string) (*coreOSRelease, error) {
 		securityFixes = append(securityFixes, fix)
 		maxCVSS = math.Max(maxCVSS, fix.CVSS)
 	}
+	return &coreOSRelease{ReleaseDate: releaseDate, ReleaseNotes: releaseNotes, SecurityFixes: securityFixes, MaxCVSS: &maxCVSS, Version: release}, nil
+}
+
+func getLatestReleaseFromJSON(m map[string]interface{}) (string, error) {
```

Comment:
I don't get the padding, leftpadding and cutpadding functions, could we have some tests (or better, [examples](https://blog.golang.org/examples)) that show how exactly to use them?

Reply:
Done.